packages/sitecore-jss-dev-tools/src/templating/plugins.ts

```diff
@@ -35,9 +35,13 @@ export interface PluginDefinition {
    */
   moduleType: ModuleType;
   /**
-   * whether to use relative or absolute paths in the generated file. By default, relative paths are used.
+   * whether to use relative or absolute paths in the generated file. By default, absolute paths are used.
```

Comment:
Why the switch to absolute by default? I think relative would be most often used

Reply:
Ah, I forgot that those aliases are absolute. The switch to absolute makes sense then given our direction to TS.